package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "fendersbot",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "nodemon ./bin/www"
```

Comment:
This is also from express-generator, but it needs to start with `./bin/www`. I've never used Heroku unfortunately!